src/running-the-course/day-4.md

```diff
@@ -23,5 +23,18 @@ Ensure that `adb sync` works with your emulator or real device and pre-build
 all Android examples using `src/android/build_all.sh`. Read the script to see
 the commands it runs and make sure they work when you run them by hand.
 
+## Async
+
+If you chose Async for Day 4 afternoon, you will need a fresh crate set up and
+the dependencies downloaded and ready to go. You can then copy/paste the
+examples into `src/main.rs` to experiment with them.
```

Comment:
I would suggest adding the large async examples to the repository as full Cargo projects (similar to how the Android chapter handles things).

That makes it trivial for an instructor to find known-good versions of the code, and we also get coverage from `cargo build` when you add those projects to the workspace for the repository.

Reply:
Thanks!